disassemblers/ofrak_pyghidra/ofrak_pyghidra/components/pyghidra_components.py

```diff
@@ -169,12 +172,22 @@ async def analyze(self, resource: Resource, config=None):
                     program_attrs = program_r.get_attributes(ProgramAttributes)
                 except NotFoundError:
                     program_attrs = await program_r.analyze(ProgramAttributes)
+
+                base_address = None
+
+                if not any(program_r.has_tag(tag) for tag in _GHIDRA_AUTO_LOADABLE_FORMATS):
+                    code_regions = await program_r.get_children_as_view(
+                        CodeRegion, r_filter=ResourceFilter.with_tags(CodeRegion)
+                    )
+                    base_address = min(r.virtual_address for r in code_regions)
```

Comment:
I don't think it's necessarily true that the minimum load address of any code region is the base address... Maybe we should be taking the min of the memory regions instead? Even that is error-prone, but probably slightly less so.